miden/tests/integration/operations/u32_ops/arithmetic_ops.rs

```diff
@@ -116,26 +116,27 @@ fn u32add_full_fail() {
 fn u32add_unsafe() {
     let asm_op = "u32add.unsafe";
 
-    // should push c = (a + b) % 2^32 onto the stack
-    // should push overflow flag d, where d = 1 if (a + b) >= 2^32 and d = 0 otherwise
+    // should push c = (a + b) % 2^32 onto the stack.
+    // should push overflow flag d, where d = 1 if (a + b) >= 2^32 and d = 0 otherwise.
     test_add_full(asm_op);
 
-    // should not fail when inputs are out of bounds
+    // should not fail when inputs are out of bounds.
     test_unsafe_execution(asm_op, 2);
 }
 
 #[test]
-fn u32addc() {
-    let asm_op = "u32addc";
+fn u32add3() {
+    let asm_op = "u32add3";
 
-    // should push d = (a + b + c) % 2^32 onto the stack, where c is 1 or 0
-    // should push overflow flag e, where e = 1 if (a + b + c) >= 2^32 and e = 0 otherwise
-    test_addc(asm_op);
+    // should push d = (a + b + c) % 2^32 onto the stack.
+    // should push overflow flag e, where e = floor(a + b + c / 2^32) if (a + b + c) >= 2^32,
+    // e = 0 otherwise.
```

Comment:
I wouldn't call `e` a flag. Let's use the following here:
```
should push e = floor((a + b + c) / 2^32) onto the stack.
```

Reply:
Actioned. Thanks.